mediator/med_io_mod.F90

```diff
@@ -119,41 +120,377 @@ logical function med_io_file_exists(vm, iam, filename)
   end function med_io_file_exists
 
   !===============================================================================
-  subroutine med_io_init()
+  subroutine med_io_init(gcomp, rc)
 
     !---------------
     ! initialize pio
     !---------------
 
+#ifdef CESMCOUPLED
     use shr_pio_mod , only : shr_pio_getiosys, shr_pio_getiotype, shr_pio_getioformat
-#ifdef INTERNAL_PIO_INIT
-    ! if CMEPS is the only component using PIO, then it needs to initialize PIO
-    use shr_pio_mod , only : shr_pio_init1, shr_pio_init2
+#else
+    use pio  , only : pio_init, pio_setdebuglevel, pio_set_rearr_opts
```

Comment:
Is this a permanent dependency on the PIO library?

Reply:
@danrosen25 there are the only functions used by the mediatory and it is permanent. Otherwise, you could not create mediator history and restart files.